### ISSUE TITLE
You are not the faction admin error

### ISSUE THREAD

**Issue by segatrooper** [ProSavage/SavageFactions#221]
**Describe the bug**
Even with factions.* and factions.admin given on luckyperms, there is still this error 
"You are not the faction admin error"
I even tried factions.admin.any

**Comment by Savag3life** [comment]
Not sure what you are looking for here.  You simply made a statement. We need more info before we can help. Are you in /f bypass? What commands are denying access, your version etc.

**Comment by srleojaco** [comment]
faction admin = faction owner?

**Comment by Drc-DEV** [comment]
Open a new issue if you still have this problem. Maybe next time tell us how to reproduce the problem step by step, the version you are using and any other info that would be relevant to the issue.
